examples/deep_dream.py

```diff
@@ -83,7 +83,8 @@ def deprocess_image(x):
 loss = K.variable(0.)
 for layer_name in settings['features']:
     # Add the L2 norm of the features of a layer to the loss.
-    assert layer_name in layer_dict.keys(), 'Layer ' + layer_name + ' not found in model.'
+    assert layer_name in layer_dict.keys(), \
```

Comment:
Please generally avoid `\` as line break. You could use parens or a if statement with a ValueError.

(Same comment applies in several other places in this PR.)

Reply:
thanks for comment.
I replaced these with parens.